presto-main-base/pom.xml

```diff
@@ -0,0 +1,602 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was wondering if we can add a checkystyle rule here to flag/prevent adding airlift HTTP namespace references in any code in `prest-main-base`

Reply:
This is a good idea. I will see if this is possible. Though, in the future after the airlift upgrade, the dependency analysis plugins will also throw an error since airlift's http dependencies will be on 17 which will not work the project bytecode level